content/source/docs/plugin/framework/types.html.md

```diff
@@ -236,10 +236,6 @@ State, Config, and Plan](/docs/plugin/framework/accessing-values.html).
 
 You may want to build your own attribute value and type implementations to allow your provider to combine validation, description, and plan customization behaviors into a reusable bundle. This helps avoid duplication or reimplementation and ensures consistency.
 
-~> **Important:** Specifying validation and plan customization for attribute
```

Comment:
I think plan customization is still not supported.

Reply:
Are you sure? There is now [`tfsdk.Attribute.PlanModifiers`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework@v0.3.0/tfsdk#Attribute.PlanModifiers), unless plan customization means something something else.

Reply:
The full quote is "Specifying validation and plan customization for attribute **types** is not yet supported." We [have that](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/attr#TypeWithValidate) for Validation, but not for plan modification. Though we do, as you rightly point out, have it for listing modifiers on a schema attribute, we can't attach plan modification to an `attr.Type` just yet.

Reply:
Ah sorry, was missing the _type_ part there. 🤦  I will add this back in with just a note about plan customization on types.